benchmarks/benchmark/tools/dlio/modules/dlio/podspec.tpl

```diff
@@ -64,6 +67,41 @@ spec:
           mountPath: ${dlio_data_mount_path}
         - name: dshm
           mountPath: /dev/shm
+
+      # remove this after the webhook is running
+      - name: gke-parallelstore-sidecar
```

Comment:
this needs to be a conditional add or we can create 2 podspec.tpl files one for PS one for GCS and pick the right one?

Reply:
removed it since webhook is up.